Assets/Scripts/Networking/PlayerManager.cs

```diff
@@ -62,12 +65,19 @@ public class PlayerManager : MonoBehaviourPunCallbacks, IPunObservable
         [SerializeField]
         private GameObject paintballPrefab;
 
+        [Tooltip("Prefab of paintball to shoot for other players")]
+        [SerializeField]
+        private GameObject paintballPrefabClient;
+
         [Tooltip("Transform the paint balls come from")]
         private Transform paintGun;
 
         //True, when the user is firing
         bool IsFiring;
 
+        //True when the shooting coroutine is running, used for fake bullets of other player
+        bool waitingToShoot = false;
+
```

Comment:
```suggestion
        //True when the shooting coroutine is running, used for fake bullets of other player
        bool shootingFake = false;

```
It would maybe be a little more understandable when used later. Cause otherwise you could also think that this is used for local user

Reply:
But as I'm reading it, doesn't it work now in a way, where only 1 bullet can be seen on screen or something? Cause I guess you can't shoot while another shot is in process?

Reply:
The comment is misleading, this is used for both user bullets. I will adjust the comment instead. You can shoot while there are multiple bullets in the air, but just need to wait for the "cooldown" to be over

Reply:
But can you see few bullets at once though?